### PR TITLE
data: PandeyHarsh433

### PR DESCRIPTION
**Pull request by PandeyHarsh433**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8086"></a>

### CODE REVIEW

File: data/PandeyHarsh433.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Harsh Pandey",
+  "bio": "Full-stack Developer | Open Source",
+  "displayStatsPublic":true,
+  "type" : "personal",
```
Comment (Vishrut19) [inline data/PandeyHarsh433.json:5]: These are not needed

**Review by Vishrut19 — CHANGES_REQUESTED** [review]

File: data/PandeyHarsh433.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    {
+      "name": "Check out my Portfolio",
+      "url": "https://pandey-harsh-c2049f.netlify.app/",
+      "icon": "FaLink"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/pa57536222",
+      "icon": "FaTwitter"
+    },
```
Comment (Vishrut19) [inline data/PandeyHarsh433.json:29]: `.` is not needed if you are not planning to add more items in the object.

File: data/PandeyHarsh433.json (hunk 1 above)
Comment (SaraJaoude) [inline data/PandeyHarsh433.json:5]: no longer required:
```suggestion
```

File: data/PandeyHarsh433.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
```
Comment (SaraJaoude) [inline data/PandeyHarsh433.json:27]: remove comma:
```suggestion
    }
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**